scanner/src/test/kotlin/scanners/fossid/FossIdUrlProviderTest.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2022 Bosch.IO GmbH
```

Comment:
Please use now

    Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)